SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamFragment.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.duosamples.about.fragments
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.text.style.ClickableSpan
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.microsoft.device.display.samples.duosamples.R
+import com.microsoft.device.display.samples.duosamples.about.addClickableLink
+import com.microsoft.device.display.samples.duosamples.databinding.FragmentAboutTeamBinding
+
+class AboutTeamFragment : Fragment(R.layout.fragment_about_team) {
+
+    companion object {
+        fun newInstance(): AboutTeamFragment = AboutTeamFragment()
+    }
+
+    private var binding: FragmentAboutTeamBinding? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentAboutTeamBinding.inflate(inflater, container, false)
+        return binding?.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupDescriptionText()
+    }
+
+    private fun setupDescriptionText() {
+        val clickableSpan = object : ClickableSpan() {
+            override fun onClick(view: View) {
+                startActivity(
```

Comment:
Also these lines can be extracted as an extension function, like this:

Activity.openUrl(url: String) {
                startActivity(
                    Intent(
                        Intent.ACTION_VIEW,
                       url
                    ).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
                )
}

Reply:
Fixed